package.json

```diff
@@ -1,6 +1,8 @@
 {
-    "name": "zh-hans.docs.eslint.org",
+    "name": "zh-hans.eslint.org",
```

Comment:
This needs to be zh-hans.docs.eslint.org because the docs need to be deployed to their own domain separate from the main site.

Reply:
I thought it would be deployed in the subdirectory same as the main site. Ok, I will revert it.